src/app/services/media.ts

```diff
@@ -108,6 +121,12 @@ export class MediaService {
       return;
     }
     this.media.splice(index, 1);
+    this.save ()
```

Comment:
This is not async method, but we don't really need to wait for this call anyway, so this is fine.